src/meta/api/src/schema_api_impl.rs

```diff
@@ -2525,6 +2533,162 @@ impl<KV: kvapi::KVApi<Error = MetaError>> SchemaApi for KV {
         Ok(CountTablesReply { count })
     }
 
+    async fn list_table_lock_revs(
+        &self,
+        req: ListTableLockRevReq,
+    ) -> Result<ListTableLockRevReply, KVAppError> {
+        let prefix = format!("{}/{}", TableLockKey::PREFIX, req.table_id);
```

Comment:
i think it is better to encapture a funciton such as table_lock_key to clarity this operation.

Reply:
> i think it is better to encapture a funciton such as table_lock_key to clarity this operation.

It's a prefix of the `TableLockKey`